schemas/com.snowplowanalytics.badrows/format_violation/jsonschema/1-0-0

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a format violation bad row",
+  "self": {
+    "vendor": "com.snowplowanalytics.badrows",
+    "name": "format_violation",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "failure": {
+      "type": "object",
+      "description": "Information regarding the format violation",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "description": "Timestamp at which the failure occurred",
+          "format": "date-time"
+        },
+        "message": {
+          "type": "string",
+          "description": "Failure message associated with the format violation",
+          "maxLength": 256
+        }
+      },
+      "required": [ "timestamp", "message" ],
+      "additionalProperties": false
+    },
+    "payload": {
+      "type": "object",
+      "description": "The collector payload that resulted in a format violation",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "description": "Timestamp at which the payload was collected",
+          "format": "date-time"
+        },
+        "headers": {
+          "type": "object",
+          "description": "List of the headers part of this collector payload"
+        },
+        "path": {
+          "type": "string",
+          "description": "Path requested by this collector payload",
+          "maxLength": 256
+        },
+        "queryString": {
+          "type": [ "string", "null" ],
+          "description": "Query string associated with this collector payload"
+        },
+        "body": {
+          "type": [ "string", "null" ],
+          "description": "Body associated with this collector payload"
+        }
+      },
+      "required": [ "timestamp", "headers", "path", "queryString", "body" ],
+      "additionalProperties": false
+    },
+    "processor": {
+      "type": "object",
+      "description": "Information about the piece of software responsible for the creation of this format violation",
+      "properties": {
+        "artifact": {
+          "type": "string",
+          "description": "Artifact responsible for the creation of this format violation",
+          "maxLength": 512
+        },
+        "version": {
+          "type": "string",
+          "description": "Version of the artifact responsible for the creation of this format violation",
+          "pattern": "^(\\d+\\.\\d+\\.\\d+.*)$",
```

Comment:
Do we need to add the option here to validate release candidates as well?

Reply:
it's covered by the `.*` at the end :+1: